src/styles/main.scss

```diff
@@ -0,0 +1,55 @@
+@mixin round ($size) {
+  width: $size;
+  height: $size;
+  border-radius: 50%;
+}
+
+.watch {
+  @include round(200px);
+  background-color: pink;
+  border: 2px solid black;
+  position: relative;
+
+  &__minutes {
+    width: 2px;
+    height: 25%;
+    background-color: black;
+    position: absolute;
+    top: 25%;
+    left: 50%;
+    z-index: 1;
+    animation: rotate 3600s steps(36, end);
+    animation-iteration-count: infinite;
+  }
+
+  &__seconds {
+    width: 2px;
+    height: 35%;
+    background-color: black;
+    position: absolute;
+    top: 15%;
+    left: 50%;
+    animation: rotate 60s steps(360, end);
+    animation-iteration-count: infinite;
```

Comment:
for minutes and seconds almost same styles. make mixin for them, and pass different styles as arguments to mixin